extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -172,7 +172,12 @@ export class Bind {
           break;
         }
         const bindings = [];
-        for (const attr of el.attributes) {
+        const attrs = el.attributes;
+        for (let j = 0;; j++) {
```

Comment:
Haha I'm not a fan either. They're in response to: https://github.com/ampproject/amphtml/pull/6967#discussion_r95473793

Hard to make everyone happy. :)